clients/wallet/src/stellar_wallet.rs

```diff
@@ -718,38 +703,32 @@ mod test {
 		let asset = StellarAsset::native();
 		let amount = 1000;
 		let request_id = [0u8; 32];
-		let correct_amount_that_should_not_fail = 100;
-		let incorrect_amount_that_should_fail = 0;
 
 		let response = wallet
 			.send_payment_to_address(
 				default_destination(),
 				asset.clone(),
 				amount,
 				request_id,
-				correct_amount_that_should_not_fail,
 				false,
 			)
 			.await;
 
 		assert!(response.is_ok());
 
-		let err_insufficient_fee = wallet
+		let tx_failed = wallet
 			.send_payment_to_address(
 				default_destination(),
 				asset.clone(),
-				amount,
+				amount + 100_000_000_000,
```

Comment:
What are we doing here? Transferring a super large amount that we cannot transfer, to force the transaction to fail? If so, maybe worth adding a comment to explain what's happening.

Reply:
i added a comment : `// forcefully fail the transaction`